internal/model/process.go

```diff
@@ -28,6 +28,7 @@ type Process struct {
 	Argv        []string
 	CommandLine string
 	Executable  string
+	Thread      *ProcessThread
```

Comment:
As commented elsewhere, should we use a non-pointer type and take the zero value to mean it should be omitted?